documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -71,6 +71,11 @@ For example:
 ----
 apiVersion: {KafkaApiVersion}
 kind: Kafka
+metadata:
+  labels:
+    app: my-cluster
+  name: my-cluster
+  namespace: myproject
```

Comment:
why we need all these fields? I guess that just `name` and `namespace` are enough for the example and note related to OpenShift routes.

Reply:
Thanks. Yes, we can drop to those.